pkg/jobstore/boltdb/store.go

```diff
@@ -421,28 +407,25 @@ func (b *BoltJobStore) getInProgressJobs(tx *bolt.Tx) ([]model.JobWithInfo, erro
 	var infos []model.JobWithInfo
 	var keys [][]byte
 
-	keys, err := b.inProgressIndex.List(tx, BucketJobsInProgress)
+	keys, err := b.inProgressIndex.List(tx)
 	if err != nil {
 		return nil, err
 	}
 
-	bktJobs := tx.Bucket(BucketJobs)
-	bktState, err := NewBucketPath(BucketPathState).Get(tx, false)
-	if err != nil {
-		return nil, err
-	}
-
-	for _, key := range keys {
+	for _, jobIDKey := range keys {
 		var job model.Job
 		var jobState model.JobState
 
```

Comment:
I don't think we are populating the job state with executions here

Reply:
Fixed in e7bdaa55